manifests/enterprise.pp

```diff
@@ -95,6 +95,13 @@
           Class['sensuclassic::package'],
         ],
       }
+
+      exec { 'sensu-enterprise-reload':
```

Comment:
perhaps a functional test that shows that the service does not get restarted when a config file changes when using the enterprise version

Reply:
I'm not sure how I'd be able to verify that. The serverspec tools for beaker can only check services are running/enabled so wouldn't be able to verify a restart didn't occur. The unit tests are only way I can think to ensure we're interacting with the Exec and not the Service.  The system's state doesn't really change between the two so acceptance tests can't tell the difference.

Verified checks changed will execute a reload:

```
Notice: /Stage[main]/Main/Node[sensu-server]/Sensuclassic::Check[check_ntp]/Sensuclassic::Write_json[/etc/sensu/conf.d/checks/check_ntp.json]/File[/etc/sensu/conf.d/checks/check_ntp.json]/content: content changed '{md5}a481e65c2316c6de3a533e6b16b12ef0' to '{md5}c99b33a6fc351ad1c4e330e5c4cd5a3b'
Notice: /Stage[main]/Sensuclassic::Enterprise/Exec[sensu-enterprise-reload]: Triggered 'refresh' from 1 event
```